pkg/services/dashboards/service/dashboard_service_integration_test.go

```diff
@@ -1,557 +0,0 @@
-package service
```

Comment:
In order to test the /api path, needed to move this to pkg/tests/api/dashboards/api_dashboards_test.go and run grafana. Note: these tests are also already covered for /apis (in pkg/tests/apis)